pyuvsim/uvsim.py

```diff
@@ -532,6 +572,22 @@ def run_uvdata_uvsim(input_uv, beam_list, beam_dict=None, catalog=None, quiet=Fa
     vis_data.Free()
 
     if rank == 0:
+        if input_order is not None:
+            if len(input_order) < 2:
+                input_order = (input_order[0], None)
+            # Don't call the functionif we're already expecting (time, baseline)
+            if input_order != ("time", "baseline"):
+                uv_container.reorder_blts(
+                    order=input_order[0],
+                    minor_order=input_order[1],
+                )
+        else:
+            warnings.warn(
+                "The parameter `blt_order` could not be identified for input_uv. "
+                "The ordering was changed to `(time, baseline)` during the simulation "
+                " but cannot be changed back."
```

Comment:
I don't like the phrase "but cannot be changed back" because it might suggest to users that blt ordering can't be changed, which of course it can be. Maybe something like this instead:
"The parameter `blt_order` could not be identified for input_uv so the ordering cannot be restored. The output will be in `(time, baseline)` order."

Reply:
I've updated the wording to be more in line with your suggestion.